helm/provisioner/values.yaml

```diff
@@ -3,17 +3,21 @@
 #
```

Comment:
the format of the new values LGTM, but we should make the new chart backward compatible with old values yaml as possible as we can, e.g.

- `daemonset.name` override the name of DaemonSet if present
- `common.configMapName` overrides the name of ConfigMap if present
- `common. podSecurityPolicy` overrides the `rbac.pspEnabled` if present
- etc.

then, people can use the old values file with the new chart and don't worry about any problems during upgrading.

Reply:
I can do this but it means duplication in the value / chart template files which is not a good approach in my eyes. 
Please have look in the helm/Readme.md i added a section about upgrading.
During tests within vagrant i saw no issues during upgrading from old to new chart version.

Reply:
ok